### PR TITLE
Add Avocent MergePoint Unity KVM Switch Support

### PR DESCRIPTION
**Pull request by mtentilucci**
Add support for Avocent MPU KVM switches. 

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [X] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/avocent-mp.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+os: avocent-mp
+text: Avocent MergePoint Unity
+type: network
+icon: avocent
+discovery:
+    -
+        sysDescr_regex:
```
Comment (Jellyfrog) [inline includes/definitions/avocent-mp.yaml:7]: It doesn't look like you need this part? Or is there other devices using same oid?

**Comment by Jellyfrog** [comment]
Maybe add a logo?

File: includes/definitions/avocent-mp.yaml (hunk 1 above)
Comment (mtentilucci) [reply includes/definitions/avocent-mp.yaml:7]: You are right, I should not need the extra sysDescr_regex. I will remove it.

File: includes/definitions/avocent-mp.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    ipSystemStats: false
+    ipmi: false
+    mpls: false
+    ntp: false
+    ospf: false
+    storage: false
+    stp: false
+    ucd-mib: false
+    ucd-diskio: false
+    wireless: false
```
Comment (Jellyfrog) [inline includes/definitions/avocent-mp.yaml:40]: you should remove these, we only force disable them if the device have troubles with them

**Review by Jellyfrog — APPROVED** [review]

**Comment by Jellyfrog** [comment]
Great job !

**Pull request merged**